it/router/RouterAuditTwoStepVerificationFeature.scala

```diff
@@ -0,0 +1,145 @@
+package router
+
+import com.github.tomakehurst.wiremock.client.WireMock._
+import com.github.tomakehurst.wiremock.client.{RequestPatternBuilder, WireMock}
+import connector.CredentialRole
+import play.api.libs.json.{JsValue, Json}
+import play.api.test.FakeApplication
+import support.page._
+import support.stubs.{CommonStubs, StubbedFeatureSpec, TaxAccountUser}
+
+import scala.collection.JavaConverters._
+import scala.collection.mutable.{Map => mutableMap}
+
+class RouterAuditTwoStepVerificationFeature extends StubbedFeatureSpec with CommonStubs {
+
+  val rule = Map(
+    "enrolments" -> "some-enrolment-category"
+  )
+
+  val locations = Map(
+    "location-1.name" -> "some-location-1",
+    "location-1.url" -> "/some-location-1",
+    "location-2.name" -> "some-location-2",
+    "location-2.url" -> "/some-location-2"
+  )
+
+
+  val optionalOid = "bbbb"  // 83.2 (percentage calculated from hashcode)
+  val mandatoryOid = "aaaa" // 4.8  (percentage calculated from hashcode)
+  val throttle = 50 // registration is mandatory if the percentage is smaller than throttle value
+
+  val additionalConfiguration = Map[String, Any](
```

Comment:
Am currently looking to see how much of this additional config can be removed from the tests